week02/MinimumWindowSubstring.cpp

```diff
@@ -0,0 +1,39 @@
+class Solution {
+public:
+    string minWindow(string s, string t) {
+        int L = 0, R = 0, minLen = INT_MAX;
+        int startingIndex = 0;
+        unordered_map<char, int> mp;
+
+        for(auto c : t) mp[c]++;
+        int cntOfExpectedChar = mp.size();
+
+        while(R < s.size()) {
+            mp[s[R]]--;
+            cntOfExpectedChar -= (mp[s[R]] == 0);
```

Comment:
```suggestion
            if (mp[s[R]] == 0) {
                cntOfExpectedChar--;
            }
```

This one is more readable in this case.

Reply:
Thanks for the recommendation.